manager/state/raft/raft.go

```diff
@@ -630,6 +632,7 @@ func (n *Node) Run(ctx context.Context) error {
 					// cancelAll, or by its own check of signalledLeadership.
 					n.wait.cancelAll()
```

Comment:
@anshulpundir do we need to put a log message just before this call to `cancelAll`?

Reply:
See log on line 615. Lemme know if you think we need another log.

Reply:
My bad, we don't. Please ignore this comment.